components/InterestZoneModal/InterestZoneView.tsx

```diff
@@ -59,29 +66,45 @@ const InterestZoneView = ({ zone, partialZone }: Props) => {
         zone={zone || partialZone!}
         form={form}
         addZone={() => {
+          if (!form.validate()) return;
           setModal(modal?.back);
           showNotification({
-            title: 'Added successfully',
-            message: 'A zone has been added!',
+            title: 'Created successfully',
+            message: 'A zone has been created!',
```

Comment:
Please confirm that the hardcoded labels above are scheduled to be moved to the translation bundle, in another PR / Jira ticket